app/models/hit_count.rb

```diff
@@ -0,0 +1,5 @@
+class HitCount < ApplicationRecord
```

Comment:
I think, hit counter should display globar user page views

Reply:
I confimed it with tanmay sir, both user and global hit counter was to be created